src/main/java/uk/gov/hmcts/reform/roleassignmentrefresh/data/GenericArrayUserType.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.hmcts.reform.roleassignmentrefresh.data;
+
+import org.hibernate.HibernateException;
+import org.hibernate.engine.spi.SharedSessionContractImplementor;
+import org.hibernate.usertype.UserType;
+
+import java.io.Serializable;
+import java.sql.Array;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Types;
+
+public class GenericArrayUserType<T extends Serializable> implements UserType {
+
```

Comment:
I don't think, we need this class. It is required for RAS database not for refresh table.